evalml/automl/automl_algorithm/iterative_algorithm.py

```diff
@@ -145,6 +145,8 @@ def _transform_parameters(self, pipeline_class, proposed_parameters):
                         component_parameters[param_name] = value.rvs(random_state=self.random_seed)
                     else:
                         component_parameters[param_name] = value
+            if name in self._pipeline_params and name == 'Drop Columns Transformer' and self._batch_number > 0:
```

Comment:
Nice. I think we'll hopefully get rid of this line soon in #2130 but I agree this is the way to get it done for now!